src/components/tx/SecurityWarnings/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { Alert, Checkbox, FormControlLabel, SvgIcon, Typography } from '@mui/material'
+import type { AlertProps } from '@mui/material'
+
+import { SecuritySeverity } from '@/services/security/modules/types'
+import AlertIcon from '@/public/images/notifications/alert.svg'
+
+import css from './styles.module.css'
+
+export const _mapSeverity = (severity: SecuritySeverity): NonNullable<AlertProps['severity']> => {
+  switch (severity) {
+    case SecuritySeverity.HIGH:
```

Comment:
We'll have to add some Severities:
This table is how redefine uses severities. I'd suggest to just copy that for now as most results will be theirs.

Severity | Number | Action
-- | -- | --
Critical | 4 | Reject this transaction
High | 3 | Funds potentially at risk, consider rejecting
Medium | 2 | Issues found, review before proceeding
Low | 1 | Minor issues found, review before proceeding
No Issues | 0

Reply:
As per our discussion this morning, I will hold off on merging this until we've had clarity on the design/legal front. cc @liliiaorlenko